examples/cube/main.rs

```diff
@@ -59,8 +62,20 @@ impl Vertex {
 }
 
 //----------------------------------------
+
+type DataType = <<ColorFormat as Formatted>::Surface as SurfaceTyped>::DataType;
```

Comment:
while it is technically correct, it's still an overkill for the examples. I think using `[u8; 4]` is fine